# request
geometry_msgs/PoseStamped start
geometry_msgs/PoseStamped end
---
# response
bool success